dataland-frontend/src/components/pages/ApiKeysPage.vue

```diff
@@ -85,9 +85,8 @@
             <div>
               <div class="text-900 font-medium text-xl text-left">API Key usage info</div>
               <div class="block text-600 mb-2 mt-4 text-left">
-                In order to use the API Key, include it into the http request header via "dataland-api-key:
-                &lt;ApiKeyValue&gt;". Alternatively it can be used like a bearer token by adding "Authorization: Bearer
-                &lt;ApiKeyValue&gt;" to the http request header.
+                You can use the API-Keys as bearer-tokens by including them in the authorization header of any requests
+                you make to dataland (i.e., "Authorization: Bearer &lt;ApiKeyValue&gt;").
```

Comment:
The value seems unneccessary, maybe confusing. I'd prefer reading "Bearer <Api Key>" or the same in lowercase